Omcodes23.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Om vataliya",
+  "bio": "I am frontend developer and exploring different technology",
+  "displayStaticPublic": true,
+  "type": "personal",
+  "avatar": "https://github.com/Omcodes23.png"
```

Comment:
Also, please remove the `type`, `displayStatsPublic`, & `avatar` keys from your `json` as we no longer need them. 

```suggestion
```